EnglishTranslation-master/SKILL.tsv

```diff
@@ -2033,11 +2033,11 @@ SKILL_20150317_002032	Strike
 SKILL_20150317_002033	Already knocked back enemy is knocked back again and receives additional damage.
 SKILL_20150317_002034	Nanta
 SKILL_20150317_002035	Reduce Knockback range of monsters when using whirlwind.
-SKILL_20150317_002036	STR increases for Melee-type attacks.
-SKILL_20150317_002037	Agility is increased. Affects ranged Physical attacks, chance of critical, and evasion.
-SKILL_20150317_002038	Physical strength improves. Max HP increases.
+SKILL_20150317_002036	Increased STR. Affects your melee physical attacks.
+SKILL_20150317_002037	Increased DEX. Affects your ranged physical attacks, critical rate and evasion.
+SKILL_20150317_002038	Increased CON. Affects your maximum HP and stamina. 
```

Comment:
There's an extra space after the period.  I'm adding this line to my PR, but without the space.